AppCenterDataStorage/AppCenterDataStorage/Internal/Model/MSPaginatedDocumentsInternal.h

```diff
@@ -7,12 +7,12 @@
 @interface MSPaginatedDocuments ()
 
 // Read-only.
-@property(nonatomic, copy, readonly) NSString *partition;
-@property(nonatomic, readonly) Class documentType;
-@property(nonatomic, readonly) MSReadOptions *readOptions;
+@property(nonatomic, copy, readonly, nullable) NSString *partition;
+@property(nonatomic, readonly, nullable) Class documentType;
+@property(nonatomic, readonly, nullable) MSReadOptions *readOptions;
 
 // Read-write (to implement pagination).
-@property(nonatomic) MSPage *currentPage;
+@property(nonatomic, nonnull) MSPage *currentPage;
```

Comment:
I would prefer to use `NS_ASSUME_NONNULL_BEGIN`/`END` as a convention instead of adding nonnull individually.  @clpolet ?

Reply:
Yes definitely, please use the `NS_ASSUME_NONNULL_BEGIN/END` pattern.